sigh/lib/sigh/options.rb

```diff
@@ -130,6 +130,11 @@ def self.available_options
                                      env_name: "SIGH_CERTIFICATE_ID",
                                      description: "The ID of the code signing certificate to use (e.g. 78ADL6LVAA) ",
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :cert_serial_number,
+                                    short_option: "-s",
+                                    env_name: "SIGH_CERTIFICATE_SERIAL_NUMBER",
+                                    description: "The Serial Number of the code signing certificate to use (e.g. 06:9f:ba:78:51:58:dc:46:d9:a6:60:3e:af:5a:13:16). Takes precedence over cert_id. For advanced use only",
```

Comment:
Most of our users are software developers, I think instead of saying `For advanced use only` it would be helpful to explain the circumstances in which they might want to match by serial number instead of cert id. (Why did you want to?)